roadmap/implementers-guide/src/node/disputes/dispute-distribution.md

```diff
@@ -29,7 +29,7 @@ This design should result in a protocol that is:
 
 #### Disputes
 
-Protocol: "/polkadot/dispute/1"
+Protocol: "/polkadot/send\_dispute/1"
```

Comment:
is the escape necessary for `_`?

Reply:
seems so. My editor highlighted it red otherwise and I checked markdown rules - it seems to be necessary. (Obviously there are different markdown implementations, so I am not 100% sure)